README.md

```diff
@@ -1,5 +1,8 @@
 # Sway Visual Studio Code Plugin
 
+![Visual Studio Marketplace Version](https://img.shields.io/visual-studio-marketplace/v/FuelLabs.sway-vscode-plugin)
```

Comment:
Should this have a marketplace link? Right now clicking just takes you to the picture of the badge. Ignore this if there's no such thing :)

Reply:
Oh yeah it should

Reply:
Fixed in [b66ca28](https://github.com/FuelLabs/sway-vscode-plugin/pull/34/commits/b66ca28cb5beec3b70d59a58b8cba82558566e79)